automatic/filezilla/update.ps1

```diff
@@ -9,8 +9,6 @@ function global:au_SearchReplace {
         }
 
         ".\legal\VERIFICATION.txt" = @{
-          "(?i)(\s+x32:).*"            = "`${1} $($Latest.URL32)"
-          "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
```

Comment:
Instead of removing these replacements, can you swap out the replacement to use replace any URL with the URL specified in the `$releases` variable?

This way, if we change this variable in the future we will also have the new download location.